dotnet-base-libraries/ReturnInsertedIdentityWithDapper/ReturnInsertedIdentityWithDapper/Methods/UseOfOutputInserted.cs

```diff
@@ -0,0 +1,25 @@
+﻿using Dapper;
+using Microsoft.Data.SqlClient;
+
+namespace ReturnInsertedIdentityWithDapper;
+
+public partial class Methods
```

Comment:
Why use partial classes in this case? You can make this simpler and just use a single class with different methods based on the approach, instead of having three different classes.

Reply:
Moved methods to one single class